static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.ts

```diff
@@ -431,6 +431,16 @@ const criteoBidder: PrebidBidder = {
 	}),
 };
 
+const smartBidder: PrebidBidder = {
+	name: 'smart',
```

Comment:
From the example [here](https://help.smartadserver.com/s/article/Header-Bidding-with-Prebid-js-client-side-integration) I think that the bidder property has to be `smartadserver`.
```suggestion
	name: 'smartadserver',
```